index.html

```diff
@@ -0,0 +1,468 @@
+                                <a href="#">Jobs</a> 
+                            </li>                             
+                            <li> 
+                                <a href="#">Contact</a> 
+                            </li>                             
+                        </ul>                         
+                    </div>                     
+                    <div class="bottom-nav"> 
+                        <ul class="hide-small-large"> 
+                            <li> 
+                                <a href="#">Latest</a> 
+                            </li>                             
+                            <li> 
+                                <a href="#">Hard Fork</a> 
+                            </li>                             
+                            <li> 
+                                <a href="#">Plugged</a> 
+                            </li>                             
+                            <li> 
+                                <a href="#">Fundamentals</a> 
+                            </li>                             
+                            <li> 
+                                <a href="#">Work 2030</a> 
+                            </li>                             
+                            <li class="hide-medium-large"> 
+                                <a href="#">Readme</a> 
+                            </li>                             
+                            <li class="hide-medium-large"> 
+                                <a href="#">Growth Quarters</a> 
+                            </li>                             
+                            <li class="hide-medium"> 
+                                <a href="#">Neural</a> 
+                            </li>                             
+                        </ul>                         
+                        <ul> 
+                            <li> 
+                                <button> 
+                                    <i class="icon i-search"></i> 
+                                </button>                                 
+                            </li>                             
+                            <li> 
+                                <button> 
+                                    <i class="icon i-menu"></i> 
+                                </button>                                 
+                            </li>                             
+                        </ul>                         
+                    </div>                     
+                    <div class="yellowBar"> 
+                        <p class="has-menu"> 
+			  Startups! You can exhibit, pitch, and meet investors at TNW2020 → </p> 
+                    </div>                     
+                </nav>                 
+            </header>             
+        </div>         
+        1              
+        <main> 
+            <div class="main-art"> 
```

Comment:
- [ ] The `<div>`s inside the `.main-art` should be `<articles>`. Remember to use semantic html. [More here.](https://bitsofco.de/sectioning-content-in-html5/)